datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -91,3 +93,16 @@ def replay_check_run(agent_collector, stub_aggregator):
 
     if runner['LastError']:
         raise Exception(runner['LastError'])
+
+
+def serialize_data(data):
```

Comment:
`deserialize_data(serialize_data(x))` fails on python3. I think that would make more sense to handle the py2/py3 compatibility layer here.